README.md

```diff
@@ -12,7 +12,7 @@ defined in the MapML (Map Markup Language)
 
 Beginning with `v0.8.2`, &lt;mapml-viewer&gt; can be loaded from the [UNPKG content delivery network](https://unpkg.com/) –
 ```html
-<script type="module" src="https://unpkg.com/@maps4html/web-map-custom-element@0.8.2/dist/mapml-viewer.js"></script>
+<script type="module" src="https://unpkg.com/@maps4html/web-map-custom-element@0.8.3/dist/mapml-viewer.js"></script>
```

Comment:
Should the README perhaps be version agnostic, and use `@latest` instead?

Reply:
Absolutely.